attachments/simple_engine/android/README.adoc

```diff
@@ -0,0 +1,67 @@
+= Android Project for Vulkan Tutorial
+
+This Android project allows you to run different chapters of the Vulkan Tutorial on Android devices.
+
+== Selecting a Chapter
+
+By default, the project builds and runs the `34_android` chapter. You can select a different chapter by setting the `chapter` property in your Gradle build.
+
+=== Available Chapters
```

Comment:
I'm confused. Does this PR add Android support to the simple game engine, or does it change existing Android support for the tutorial chapters 34 and 35 (which already work on Android)?